Description: Turn off annoying warning when GNOME Fallback mode is run
Forwarded: no
Author: Jeremy Bicha <jbicha@ubuntu.com>
Index: gnome-session-3.5.4/gnome-session/gsm-manager.c
===================================================================
--- gnome-session-3.5.4.orig/gnome-session/gsm-manager.c	2012-07-19 13:53:01.315382481 +1200
+++ gnome-session-3.5.4/gnome-session/gsm-manager.c	2012-07-19 13:53:04.427382372 +1200
@@ -1617,7 +1617,6 @@
                 do_phase_startup (manager);
                 break;
         case GSM_MANAGER_PHASE_RUNNING:
-                possibly_show_fallback_dialog (manager);
                 g_signal_emit (manager, signals[SESSION_RUNNING], 0);
                 update_idle (manager);
                 break;
